dev/fattest.simplicity/src/componenttest/annotation/SkipIfSysProp.java

```diff
@@ -81,6 +81,7 @@
     // OS system properties
     public static final String OS_ZOS = "os.name=z/OS";
     public static final String OS_IBMI = "os.name=OS/400";
+    public static final String JAVA = "os.name=ISERIES";
```

Comment:
It's looking good. Thanks for your work. Can you rename this to something more specific like OS_ISERIES? This will keep the pattern already established. Also before merging, can you try running a custom SOE Build?